app.js

```diff
@@ -0,0 +1,40 @@
+require('dotenv').config();
+
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const routes = require('./routes');
+const appDataSource = require('./models/appDataSource');
+const { globalErrorHandler } = require('./utils/error');
```

Comment:
```suggestion
const express = require('express');
const cors = require('cors');
const morgan = require('morgan');

const routes = require('./routes');
const appDataSource = require('./models/appDataSource');
const { globalErrorHandler } = require('./utils/error');
```
외부 모듈과, 커스텀 모듈의 구분을 지어주면 좋을 것 같습니다~